clients/include/testing_matmul.hpp

```diff
@@ -1721,7 +1721,7 @@ void testing_matmul_with_bias(const Arguments& arg,
 
     // set workspace
     device_vector<unsigned char>* dWorkspace     = nullptr;
-    size_t                        workspace_size = 0;
+    size_t                        workspace_size = arg.user_allocated_workspace;
```

Comment:
workspace_size is only for recording the current max required workspace size.
If you want to change the max allowed workspace size, you should modify max_workspace_size as arg.user_allocated_workspace.

Reply:
Done, thanks.

Reply:
Assigning max_workspace_size to arg.user_allocated_workspace will make gtest fail, so still using the oldest change.